data_steward/deid/config/ids/tables/procedure_occurrence.json

```diff
@@ -1,8 +1,20 @@
         }
```

Comment:
not finding alias `procedure_occurence` in `procedure_occurrence.procedure_occurrence_id = record2.procedure_occurrence_id`